.github/workflows/build-images-for-tag-release.yaml

```
@@ -1,9 +1,12 @@
 name: Build and Publish Images For Tag Release
 
 on:
+  workflow_dispatch:
   push:
     tags:
       - "v[0-9]+.[0-9]+.[0-9]+-?**"
+      - 'v[0-9]+.[0-9]+.[0-9]+-[a-zA-Z]*'
```

```suggestion
      - "v[0-9]+.[0-9]+.[0-9]+"
      - "v[0-9]+.[0-9]+.[0-9]+-*"
```

semantic versioning:
```
<valid semver> ::= <version core>
                 | <version core> "-" <pre-release>
                 | <version core> "+" <build>
                 | <version core> "-" <pre-release> "+" <build>

<version core> ::= <major> "." <minor> "." <patch>
```